icons/bowling.svg

```diff
@@ -0,0 +1,18 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M20.5 4.5a2.5 2.5 0 0 0-5 0C15.5 6 16 6 16 9c0 2-2 4-2 7 0 1 1 6 3 6h2c2 0 3-5 3-6 0-3-2-5-2-7 0-3 .5-3 .5-4.5" />
+  <path d="M16 9h4" />
+  <path d="M11.7 6.2c-.6-.1-1.1-.2-1.7-.2a8 8 0 0 0 0 16c.5 0 1-.1 1.5-.1" />
```

Comment:
This path has some very funky segments:
![image](https://github.com/lucide-icons/lucide/assets/17746067/636259d8-f997-4fb5-8994-7a3652cfa556)

A very manual fix:
```suggestion
  <path d="M11.65 6.17a8 8 0 1 0-.27 15.71" />
```